src/Framework/ContaoFramework.php

```diff
@@ -203,11 +205,11 @@ private function setConstants()
      */
     private function getMode()
     {
-        if ($this->isBackendScope()) {
+        if ($this->request && $this->isBackendScope($this->request)) {
```

Comment:
`$this->request` should not be necessary, because `isFrontendScope()` will handle `null`, right?